packages/inter-protocol/src/proposals/startPSM.js

```diff
@@ -79,11 +79,19 @@ export const startPSM = async (
       E(E(zoe).getInvitationIssuer()).getAmountOf(poserInvitationP),
     ]);
 
+  const [anchorInfo, stableInfo] = await Promise.all(
```

Comment:
s/stable/minted

Reply:
okay.  It starts from `STABLE.symbol`, but I'll rename it locally.